lib/datadog/tracing/contrib/rack/middlewares.rb

```diff
@@ -139,7 +139,7 @@ def set_request_tags!(trace, request_span, env, status, headers, response, origi
             request_span.set_tag(Tracing::Metadata::Ext::TAG_KIND, Tracing::Metadata::Ext::SpanKind::TAG_SERVER)
 
             if status != 404 && (last_route = trace.get_tag(Tracing::Metadata::Ext::HTTP::TAG_ROUTE))
-              last_script_name = trace.get_tag(Tracing::Metadata::Ext::HTTP::TAG_ROUTE_PATH).to_s
+              last_script_name = trace.get_tag(Tracing::Metadata::Ext::HTTP::TAG_ROUTE_PATH).to_s || ''
```

Comment:
to_s will always be truthy I think so the `|| ''` wouldn't do anything?

Reply:
fixed: [7317018](https://github.com/DataDog/dd-trace-rb/pull/3849/commits/7317018fe09188035e5f7799603c0d46b5a95912)